.circleci/run_browserstack_acceptance.sh

```diff
@@ -5,4 +5,14 @@ export BROWSERSTACK_BUILD_ID="${CIRCLE_BRANCH} - ${CIRCLE_BUILD_NUM}"
 COMMIT_MSG_TITLE=$(git log -n 1 --pretty=format:%s)
 export BROWSERSTACK_TEST_RUN_NAME=$COMMIT_MSG_TITLE
 
-npx testcafe "browserstack:ie@11.0,browserstack:safari" tests/acceptance/acceptancesuites/*.js -q
\ No newline at end of file
+if [[ $CIRCLE_BRANCH == release/*
+  || $CIRCLE_BRANCH == hotfix/*
+  || $CIRCLE_BRANCH == master
+  || $CIRCLE_BRANCH == support* ]]
+then
+  npx testcafe "browserstack:ie@11.0,browserstack:safari" tests/acceptance/acceptancesuites/*.js \
+  -q --app "npx serve -l tcp://0.0.0.0:9999" --app-init-delay 4000
```

Comment:
oh my god is this running the server in the background? I could not figure out how to do this

Reply:
would it make sense to use a config file for this since it's a little lengthy? (and repeated in a few places)

Reply:
Yeah! --app executes the given command and we can specify the wait time for the application to setup before starting the tests. I updated to have a config file for some repetitive parts of the command line, not sure if that's what you have in mind.